### DISCUSSION TITLE
Error when uploading images

### DISCUSSION THREAD

**Discussion by wrux** [sanity-io/sanity#3596] [Sanity Studio Support] [answered]
I’m using the latest version of Studio 3 and keep getting an `Eve is not a function` error when trying to upload images. Any ideas here?

<img width="423" alt="Screenshot 2022-09-02 at 12 08 07" src="https://user-images.githubusercontent.com/1453384/188350281-63a03d35-1775-4b2d-ab66-9110dcd1334c.png">

<img width="494" alt="Screenshot 2022-09-02 at 12 08 17" src="https://user-images.githubusercontent.com/1453384/188350285-3158ca9b-c217-485f-9159-350f64a7c55d.png">

**Comment by AmeyGohil** [comment, marked as answer]
Hey @wrux, I was facing the same issue, and found a solution in discussions - [3411](https://github.com/sanity-io/sanity/discussions/3411)
This is fixed in the release [v3.0.0-dev-preview.12](https://github.com/sanity-io/sanity/releases/tag/v3.0.0-dev-preview.12)
